author/backends.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
PEP8/flake8: django-author/author/__init__.py:2:1: E265 block comment should start with '# '